packages/cli/src/ceramic-daemon.ts

```diff
@@ -242,6 +242,8 @@ type AdminApiModelIdsMutationMethod = {
   method: (modelsIDs: Array<StreamID>) => Promise<void>
 }
 
+const MODELS_DEPRECATION_AT = 'Thu, 29 Jun 2023 23:59:59 GMT'
```

Comment:
why does the timestamp of when it was deprecated matter to anyone?

Reply:
I think a more informative message like "the /models HTTP endpoint has been deprecated in favor of the /modelData endpoint" would be more useful

Reply:
This is the format  specified for the header. https://datatracker.ietf.org/doc/html/draft-ietf-httpapi-deprecation-header-02#page-3